CHANGELOG.md

```diff
@@ -2,6 +2,86 @@
 
 ## Unreleased
 
+This release represents the release candidate ("RC") release for the Logs Bridge API / SDK. In the
+following release, `opentelemetry-api-logs` will be merged into `opentelemetry-api`,
```

Comment:
(I initially read "following release" as the release that is described in the following text)

```suggestion
next release, `opentelemetry-api-logs` will be merged into `opentelemetry-api`,
```